### DISCUSSION TITLE
Contrast & NextJs

### DISCUSSION THREAD

**Discussion by santbob** [vercel/next.js#38244] [Help]
We are using [Contrast](https://docs.contrastsecurity.com/en/node-js-agent.html) for their security services. it requires us to start node-based applications `node-contrast index.js` which is similar to `node index.js`. `node_contrast` acts as a proxy for `node` command and inturns start the app. 

How can I do the same thing to start a nextjs application as nextjs uses the `next` command instead of `node` and contrast currently doesn't have support for nextjs style natively.

kindly help

**Comment by wwwdepot** [comment]
It might work with https://nextjs.org/docs/advanced-features/output-file-tracing#automatically-copying-traced-files.
